src/core/network.js

```diff
@@ -115,20 +133,17 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if (GENERIC || FIREFOX || MOZCENTRAL)
+      if (args.onProgressiveData && supportsMozChunked) {
+        xhr.responseType = 'moz-chunked-arraybuffer';
+        pendingRequest.onProgressiveData = args.onProgressiveData;
+        pendingRequest.mozChunked = true;
       } else {
         xhr.responseType = 'arraybuffer';
       }
+//#else
```

Comment:
The `supportsMozChunked` is always true for MOZCENTRAL and FIREFOX, but it can also be true or false for non-FF/CHROMIUM. Because of this, and because we don't want preprocessor magic to eliminate the boolean, the `supportsMozChunked` check always happens in non-Chromium builds.
The appearance of the preprocessor special case for Chromium can be removed by adding

```
//#if CHROMIUM
//var supportsMozChunked = false;
//#endif
```

but that doesn't gain us much, does it?

Do you have a preference for any of the presented alternatives (this comment and my previous comment)?